packages/extension-browser/src/devtools/panel/utils/inspire.ts

```diff
@@ -0,0 +1,25 @@
+const messages = [
+    `78.6% of websites use a JS library with known vulnerabilities`,
+    `The average load time for mobile sites is 19 seconds over 3G connections`,
+    `53% of mobile site visits are abandoned if pages take longer than 3 seconds to load`,
+    `84% of websites perform a redirect for at least one resource`,
+    `97.5% of sites forget to compress one or more resources`,
+    `72.8% of sites compress at least one resource they shouldn't`,
+    `52% of resources aren't cacheable`,
+    `The median site size is 1,731KB`,
+    `1MB of JavaScript takes more time to process than 1MB of a JPEG file`
+];
+
+let remaining = [...messages];
+
+const inspire = (): string => {
+    if (!remaining.length) {
+        remaining = [...messages];
```

Comment:
Nice catch

Reply:
At least the earlier version didn't run out of messages as it always selected a random index without mutating the array. It just suffered from the possibility of repeating the same message before getting through the list, potentially even back-to-back.